Symfony/CS/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -24,9 +24,10 @@ class InvalidConfigurationException extends \InvalidArgumentException
 {
     /**
      * @param string $message
+     * @param int    $code
```

Comment:
> another down side of hiding the default form the signature

I don't agree that it's a downside at all. Infact, I see it as better that the alternative.